index.bs

```diff
@@ -201,13 +201,16 @@ The request has an <dfn>associated element</dfn>, the {{Element}} causing this i
 
 NOTE: we assume that there is one <a>image request</a> for each {{Element}} that a <a>background-image</a> property affects and for each URL that the <a>background-image</a> property specifies. So, for example, if there is a style with two URLs affecting all <{div}>s, and there are two <{div}>s, then there will be four <a>image requests</a>. This means that a single <a>background-image</a> property could produce multiple {{PerformanceElementTiming}} entries because it can affect multiple elements and because it can specify multiple URLs.
 
+Every image resource that is obtained from a data URL [[RFC2397]] has an associated <a>image request</a> which is not fetched but still needs to be loaded and decoded.
+This request also has an <a>associated element</a> which caused this image resource.
+
 Each {{Document}} has <dfn>images pending rendering</dfn>, a list of <a>image requests</a> which are considered loaded and decoded but not yet rendered.
 When an <a>image request</a> has become <a>completely available</a> and decoded, run the following steps:
 
 <div algorithm="image element loaded">
     1. Let |imageRequest| be the input <a>image request</a>.
     1. Let |element| be |imageRequest|'s <a>associated element</a>.
-    1. If the <a>timing allow check</a> fails for |imageRequest|'s resource, then:
+    1. If |imageRequest| is not a data URL [[RFC2397]] and if the <a>timing allow check</a> fails for |imageRequest|'s resource, then:
```

Comment:
Ideally, we'd have the timing allow check handle that case (after its Fetch integration... sigh...). I'm not sure what origin are data URIs though.

@annevk - are the same-origin as the document that includes them? Another origin?

Reply:
They are same-origin for images.

Reply:
It'd be better to define this on top of Fetch at which point it'd be clear...

Reply:
No arguments here, just prioritization...

Reply:
Given the rather significant issues that have arisen from this approach in the past "just prioritization" is a rather unsatisfactory dismissal.

Reply:
This is far from being a dismissal. It's on my TODO list and something I'm constantly working towards (e.g. by reducing the number of open issues on current Resource Timing and Navigation Timing specs, so that we'd have a clean slate of spec and tests to enable us to rewrite both based on Fetch). It's just not the *only* thing on my TODO list.

I'd more than welcome any resources Mozilla could send towards that goal.

Reply:
Timing allow origin check currently seems to have a fetch request as input. Do data URIs have an associated fetch request or not? Also, ResourceTiming explicitly says that data URIs are not exposed because they're not fetched, hence why I special cased it. Also this website says that data URIs are treated as opaque origins, is that just for scripts then and the website should be updated?
https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs

Reply:
data URIs have fetched (handled by [scheme fetch](https://fetch.spec.whatwg.org/#scheme-fetch) AFAICT), so it seems like ResourceTiming is wrong.

Given @annevk's response and Fetch's processing related to origin ([request origin set to request's client's origin](https://fetch.spec.whatwg.org/#ref-for-concept-request-origin%E2%91%A7), then doesn't get overridden AFAICT), it seems like MDN is wrong as well.

Reply:
Ok, still I'm unsure if ResourceTiming handles this right now so I rather make this explicit here.